vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -193,52 +200,54 @@ type internal FSharpCheckerWorkspaceServiceFactory
                 member this.Checker = checkerProvider.Checker
                 member this.ProjectInfoManager = projectInfoManager }
 
-type
-    [<Guid(FSharpCommonConstants.packageGuidString)>]
-    [<ProvideLanguageService(languageService = typeof<FSharpLanguageService>,
-                             strLanguageName = FSharpCommonConstants.FSharpLanguageName,
-                             languageResourceID = 100,
-                             MatchBraces = true,
-                             MatchBracesAtCaret = true,
-                             ShowCompletion = true,
-                             ShowMatchingBrace = true,
-                             ShowSmartIndent = true,
-                             EnableAsyncCompletion = true,
-                             QuickInfo = true,
-                             DefaultToInsertSpaces = true,
-                             CodeSense = true,
-                             DefaultToNonHotURLs = true,
-                             EnableCommenting = true,
-                             CodeSenseDelay = 100,
-                             ShowDropDownOptions = true)>]
-    internal FSharpPackage() =
+
+[<Guid(FSharpCommonConstants.packageGuidString)>]
+[<ProvideLanguageService 
+    (   languageService = typeof<FSharpLanguageService>
+    ,   strLanguageName = FSharpCommonConstants.FSharpLanguageName
```

Comment:
Please no commas at start of line. Please stick to the coding conventions already in use in the codebase